README.md

```diff
@@ -1,2 +1,63 @@
-# authentication
-Ruby on Rails authentication project for learning purposes
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Authentication project
+
+> This is a project for users authentication using Rails and Devise.
+> In this project, I created a simple database for Users and Posts, where the data will be stored for learning purposes.
+> This project helps me to understand better how Devise works with Rails.
+
+![screenshot](/screenshot.jpg)
+
+> The page above is the index page shown to users who are not logged in yet.
+
+![screenshot](/screenshot2.jpg)
+
+> The page above is the login page.
+
+![screenshot](/screenshot3.jpg)
+
+> The page above is the index page shown to users after authentication.
+
+![screenshot](/screenshot3.jpg)
+
+> The page above is the user registration page.
+
+## Built With
+
+- Ruby
+- Ruby on Rails
+- PostgreSQL
+- Devise
+- Bulma CSS
+
+# How to use
+
+- Make sure you have Ruby and Rails installed on your computer. If you don't have it, simply install [Ruby](https://www.ruby-lang.org/pt/documentation/installation/), and follow this tutorial to install [rails](https://gorails.com/setup/ubuntu/20.10).
+
+- Then, clone this repository to your computer.
```

Comment:
Specify in details how the user can clone your repo. Bear in mind that some people are not familiar with coding/github so it must be detailed